tools/RAiDER/utilFcns.py

```diff
@@ -100,51 +98,47 @@ def ecef2enu(xyz, lat, lon, height):
     return np.stack((e, n, u), axis=-1)
 
 
-def gdal_extents(fname):
+def rio_extents(fname):
     if os.path.exists(fname + '.vrt'):
         fname = fname + '.vrt'
```

Comment:
This may not be needed anymore. We were facing an issue where gdal wouldn't read certain attributes from the .dat file but would from the .vrt file. I think it was mainly the NoDataValue. I don't think rasterio has this limitation; @cmarshak do you happen to know?